### PR TITLE
Nitroglycerin now can explode in cigs and E-cig bombs are more consistent

### PR DESCRIPTION
**Pull request by ghost**
### Intent of your Pull Request

Nitroglycerin can now be used to explode cigs due to its explosive nature, also E-cigarettes now have the same kind of explosive property as regular cigs. 

#### Changelog

:cl:  
rscadd: Nitroglycerin now create better cig bombs, also e-cigs now have the same explosive properties of regular cigs.  

/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
A single cig full of nitro is an instant gib,im kinda eh on this as nitro does require botany or xeno to help but its retardedly strong

File: code/game/objects/items/cigs_lighters.dm [hunk 1]
```diff
@@ -863,6 +869,18 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 					e.start()
 					qdel(src)
 				return
+				if(reagents.get_reagent_amount(/datum/reagent/fuel)) // the fuel explodes, too, but much less violently
```
Comment (alexkar598) [inline code/game/objects/items/cigs_lighters.dm:872]: Unreachable code as the line before ends the proc

**Review by alexkar598 — DISMISSED** [review]

File: code/game/objects/items/cigs_lighters.dm (hunk 1 above)
Comment (AsV9) [reply code/game/objects/items/cigs_lighters.dm:872]: nice

File: code/game/objects/items/cigs_lighters.dm [hunk 2]
```diff
@@ -197,6 +197,12 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 		e.start()
 		qdel(src)
 		return
+	if(reagents.get_reagent_amount(/datum/reagent/nitroglycerin)) // nitrogylcerin explodes with a whole lot more strength
+		var/datum/effect_system/reagents_explosion/e = new()
+		e.set_up(round(reagents.get_reagent_amount(/datum/reagent/nitroglycerin) / 1, 1), get_turf(src), 0, 0)
```
Comment (Amelia0010) [inline code/game/objects/items/cigs_lighters.dm]: you're dividing by 1? that seems kinda redundant.

File: code/game/objects/items/cigs_lighters.dm [hunk 3]
```diff
@@ -862,7 +868,19 @@ CIGARETTE PACKETS ARE IN FANCY.DM
...
+					return
+				if(reagents.get_reagent_amount(/datum/reagent/fuel)) // the fuel explodes, too, but much less violently
+					var/datum/effect_system/reagents_explosion/e = new()
+					e.set_up(round(reagents.get_reagent_amount(/datum/reagent/fuel) / 5, 1), get_turf(src), 0, 0)
+					e.start()
+					qdel(src)
+					return
+				if(reagents.get_reagent_amount(/datum/reagent/nitroglycerin)) // nitrogylcerin explodes with a whole lot more strength
+					var/datum/effect_system/reagents_explosion/e = new()
+					e.set_up(round(reagents.get_reagent_amount(/datum/reagent/nitroglycerin) / 1, 1), get_turf(src), 0, 0)
```
Comment (Amelia0010) [inline code/game/objects/items/cigs_lighters.dm]: again, dividing by 1 seems redundant

**Review by Amelia0010 — DISMISSED** [review]

**Pull request merged**